lib/cocoapods/installer/installation_options.rb

```diff
@@ -181,6 +181,13 @@ def hash
       # since the previous installation.
       #
       option :incremental_installation, false
+
+      # Whether to create a `Pods.xcodeproj` or not
```

Comment:
missing period at the end of this.

Reply:
actually just add this as comment to match the above style of comments. This documentation will appear in the guides website.

```ruby
      # Whether to skip generating the `Pods.xcodeproj` and perform only dependency resolution and downloading.
      #
```